index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+
+  <body>
+    <header>
+      <a id="logo">Farida's Logo</a>
+      <img src="./images/Union.jpg" alt="menue bar" id="menue-button">
+    </header>
+    <section id="intro">
+      <h2>I'm Farida Glad to see you!</h2>
+      <p id="bio">I’m a software developer! I can help you build a product, 
+        feature or website Look through some of my work and experience! 
+        If you like what you see and have a project you need coded, 
+        don’t hestiate to contact me.</p>
+      <h3>let's connect</h3>
+      <section id="social">
+        <img src="./images/Vector.png" alt="twitter">
+        <img src="./images/Linkedin-icon.jpg" alt="linkedin">
+        <img src="./images/Vector.jpg" alt="email">
+        <img src="./images/Vector-2.jpg" alt="github">
+        <img src="./images/Vector-3.jpg" alt="angel-list">
+      </section>
+    </section>
```

Comment:
- [ ] Kindly increase the spacing at the top of this section to match the Figma design.

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/164272283-d48ad59f-6942-4510-b250-ec688ab59e55.png) | ![image](https://user-images.githubusercontent.com/24703171/164272752-cf0c408d-c1e7-4f5b-8150-29a7d3da7662.png) |